autopush/websocket.py

```diff
@@ -257,8 +307,55 @@ def message(self):
 
     @property
     def user_agent(self):
+        # type: () -> str
         return self._user_agent or "None"
 
+    @property
+    def reset_uaid(self):
+        # type: () -> bool
+        return self._reset_uaid
+
+    @reset_uaid.setter
+    def reset_uaid(self, value):
+        if value:
+            self._reset_uaid = True
```

Comment:
kind of annoying to keep this and other things in 2 places, can PushState be solely responsible and insert it into the logging some other way?